packages/govuk-frontend/src/govuk/components/accordion/accordion.yaml

```diff
@@ -48,30 +48,45 @@ params:
     required: true
     description: An array of sections within the accordion.
     params:
-      - name: heading.text
-        type: string
+      - name: heading
+        type: object
         required: true
-        description: The title of each section. If `heading.html` is supplied, this is ignored.
-      - name: heading.html
-        type: string
-        required: true
-        description: The HTML content of the header for each section. Used as the title for each section. The header is inside the HTML `<button>` element, so you can only add [phrasing content](https://html.spec.whatwg.org/#phrasing-content) to it.
-      - name: summary.text
-        type: string
-        required: false
-        description: Text content for summary line. If `summary.html` is supplied, this is ignored.
-      - name: summary.html
-        type: string
+        description: The heading of each section.
+        params:
+          - name: text
+            type: string
+            required: true
+            description: If `html` is set, this is not required. The heading text of each section. If `html` is provided, the `text` option will be ignored.
+          - name: html
```

Comment:
Q: why do some of them have "params:" and others do not?

Reply:
There's always the component `params:` at the top of these files

But components often have sections inside them like **Accordion** items

Then each section has its own child sections like heading, summary, content with their own `params:`

The deeper it goes, the more **Nunjucks macro options** links + tables get generated
https://design-system.service.gov.uk/components/table/#options-table-example--primary

<img width="802" alt="Table params with links" src="https://github.com/alphagov/govuk-frontend/assets/415517/1de729f7-ae57-4fdb-9730-d548b5dacda7">